src/lib/protocols/proof/messages/ProofRequest.ts

```diff
@@ -0,0 +1,41 @@
+export class ProofRequest {
```

Comment:
```suggestion
export class ProofRequestMessage {
```

Also update the file name please!

Reply:
Done